tests/integration/test_installation.py

```diff
@@ -537,24 +539,31 @@ def test_table_migration_job(  # pylint: disable=too-many-locals
     # assert the workflow is successful
     assert workflows_install.validate_step("migrate-tables")
     # assert the tables are migrated
-    assert ws.tables.get(f"{dst_catalog.name}.{dst_schema.name}.{src_managed_table.name}").name
-    assert ws.tables.get(f"{dst_catalog.name}.{dst_schema.name}.{src_external_table.name}").name
-    # assert the cluster is configured correctly
-    install_state = installation.load(RawState)
-    job_id = install_state.resources["jobs"]["migrate-tables"]
-    for job_cluster in ws.jobs.get(job_id).settings.job_clusters:
-        cluster_spec = job_cluster.new_cluster
-        assert cluster_spec.autoscale.min_workers == 2
-        assert cluster_spec.autoscale.max_workers == 20
-        assert cluster_spec.spark_conf["spark.sql.sources.parallelPartitionDiscovery.parallelism"] == "1000"
+    try:
+        assert ws.tables.get(f"{dst_catalog.name}.{dst_schema.name}.{src_managed_table.name}").name
+        assert ws.tables.get(f"{dst_catalog.name}.{dst_schema.name}.{src_external_table.name}").name
+    except NotFound:
+        assert (
+            False
+        ), f"{src_managed_table.name} and {src_external_table.name} not found in {dst_catalog.name}.{dst_schema.name}"
+
+    # skip asserting following until we fix the prompt to ask for these values in another PR
+    # # assert the cluster is configured correctly
+    # install_state = installation.load(RawState)
+    # job_id = install_state.resources["jobs"]["migrate-tables"]
+    # for job_cluster in ws.jobs.get(job_id).settings.job_clusters:
```

Comment:
Remove commented code. It would lead to more commented code and we wouldn't know why it's there

Reply:
This will be temporary, I'm working on another PR that will bring these codes back.